dashboard/app/controllers/course_offerings_controller.rb

```diff
@@ -25,7 +25,8 @@ def update
   def quick_assign_course_offerings
     return head :forbidden unless current_user
 
-    offerings = QuickAssignHelper.course_offerings(current_user, request.locale)
+    participant_type = params[:participantType] || Curriculum::SharedCourseConstants::PARTICIPANT_AUDIENCE.student
```

Comment:
Do we know when this second case will happen? Or is this a failsafe and we are always expecting a participantType param here?

Reply:
Good question -- I added it as a failsafe but now I'm wondering if we should reject the request if there isn't a participant type 🤔

Reply:
I decided to reject the request. We can always loosen this restriction in the future if needed